main.js

```diff
@@ -36,6 +36,48 @@ var formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+function ODate(rootEl) {
```

Comment:
This is a bit academic.  When exported, the constructor can be called anything you like, and if you export it via the build service it will be `Origami['o-date']`.